tools/clang/test/DXILValidation/val-dx-type-lowprec.ll

```diff
@@ -0,0 +1,145 @@
+; RUN: %dxv %s
+
```

Comment:
It would be nice to add a comment to the test stating its purpose since it is not checking for any expected output, but just making sure it validates successfully it makes it hard to deduce what exactly is being tested here.

Reply:
Oh it looks like there is a comment in the hlsl. Maybe just a pointer to that file would be helpful? Not sure if this is a common pattern in the tests in this directory that would make it obvious to go look at the hlsl.

Reply:
I'll comment pointing to the HLSL.